smt/expr.cpp

```diff
@@ -1447,6 +1447,35 @@ expr expr::mkUF(const char *name, const vector<expr> &args, const expr &range) {
   return Z3_mk_app(ctx(), decl, num_args, z3_args.data());
 }
 
+expr expr::mkRecFnApp(const char *name, const vector<expr> &argdefs,
```

Comment:
maybe change the name to mkRecFn, since this doesn't create an application, just the fn decl